### PR TITLE
[feature][connector][hudi] add hudi json read function and sql read/write

### PR DESCRIPTION
**Pull request by yc576192**
The Hudi Connector based on Hudi version 0.13 provides JSON read and sql read/write capabilities

### CODE REVIEW

**Comment by PaddyMelody** [comment]
approve sql read/writer

**Comment by PaddyMelody** [comment]
About hudi json read

hudiReader will produce org.apache.hudi.table.data.ColumnarRowData ,but the type required by all ChunjunWriters is com.dtstack.chunjun.element.ColumnRowData

File: chunjun-connectors/chunjun-connector-hudi/src/main/java/com/dtstack/chunjun/connector/hudi/converter/HudiRowDataMapping.java [hunk 1]
```diff
@@ -17,16 +17,27 @@
...
 
 import org.apache.flink.table.api.DataTypes;
 import org.apache.flink.table.types.DataType;
 import org.apache.flink.table.types.logical.BinaryType;
 
+import java.util.Locale;
+
 public class HudiRowDataMapping {
-    public static DataType apply(TypeConfig type) throws UnsupportedTypeException {
-        switch (type.getType()) {
```
Comment (FlechazoW) [inline chunjun-connectors/chunjun-connector-hudi/src/main/java/com/dtstack/chunjun/connector/hudi/converter/HudiRowDataMapping.java:29]: Why not use typeConfig

File: chunjun-connectors/pom.xml [hunk 2]
```diff
@@ -106,7 +106,7 @@
 
 		<!-- nebula -->
 		<module>chunjun-connector-nebula</module>
-        <module>chunjun-connector-kingbase</module>
+        <module>chunjun-connector-hudi</module>
```
Comment (FlechazoW) [inline chunjun-connectors/pom.xml:109]: Please add hudi connector, rather than remove kingbase connector.

**Comment by yc576192** [comment]
ok

**Comment by cs3163077** [comment]
No data for sqlservercdc to hudi

**Pull request merged**